src/scout_apm/django/apps.py

```diff
@@ -17,8 +19,8 @@ class ScoutApmDjangoConfig(AppConfig):
     verbose_name = "Scout Apm (Django)"
 
     def ready(self):
-        # Copy django configuration to scout_apm's config
-        ConfigAdapter.install()
+        self.update_scout_config_from_django_settings()
+        setting_changed.connect(self.on_setting_changed)
```

Comment:
Double checking - this settings watcher works on older versions of Django?

Reply:
Yes the signal has been there since at least 1.8 which is what we support ( https://github.com/django/django/blob/stable/1.8.x/django/core/signals.py#L6 ) and tests will check it works too